docs/installation/advanced/integrating-from-source-webpack.md

```diff
@@ -253,12 +253,13 @@ If you want to use TypeScript, add the `tsconfig.json` file at the root of your
 {
   "compilerOptions": {
     "lib": [
-      "ESNext",
-      "dom"
+      "DOM",
```

Comment:
Why do we show TS configuration? Do we have any specific requirements for ts-config to make CKE work?

If not: ts-config is none of our business.

Reply:
Makes sense to limit it and have it consistent with the upgrade guide.